sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/continuous/EpochCoordinator.scala

```diff
@@ -224,24 +227,24 @@ private[continuous] class EpochCoordinator(
         partitionOffsets.collect { case ((e, _), o) if e == epoch => o }
       if (thisEpochOffsets.size == numReaderPartitions) {
         logDebug(s"Epoch $epoch has offsets reported from all partitions: $thisEpochOffsets")
-        query.addOffset(epoch, stream, thisEpochOffsets.toSeq)
+        queryExecution.addOffset(epoch, stream, thisEpochOffsets.toSeq)
         resolveCommitsAtEpoch(epoch)
       }
       checkProcessingQueueBoundaries()
   }
 
   private def checkProcessingQueueBoundaries() = {
-    if (partitionOffsets.size > epochBacklogQueueSize) {
-      query.stopInNewThread(new IllegalStateException("Size of the partition offset queue has " +
-        "exceeded its maximum"))
+    if (partitionOffsets.size > epochMsgBacklogQueueSize) {
```

Comment:
As I see you are reinterpret the new config `CONTINUOUS_STREAMING_EPOCH_MSG_BACKLOG_QUEUE_SIZE` in place of the old `CONTINUOUS_STREAMING_EPOCH_BACKLOG_QUEUE_SIZE `.

I think this is can be done (if good reasons supports it) as the old config was only introduced into Spark 3.0.0-SNAPSHOT in the PR https://github.com/apache/spark/pull/23156.

Reply:
Hmm... You remind me, we'd better to use a new config even though `CONTINUOUS_STREAMING_EPOCH_BACKLOG_QUEUE_SIZE ` is a new introduced config in Spark 3.0.0-SNAPSHOT.